File: PHP/PbusquedaMVC/Modelo/Conexion.php
<?php

class Conexion
{
    public function conectar()
    {
        $server = "localhost";
        $user = "root";
        $psw = "";
        $bd = "repasos";

        $con = mysqli_connect($server, $user, $psw, $bd);

        if (!$con) {
            die("ERROR" . mysqli_connect_error());
        } else {
            return $con;
        }
    }
}
?>

File: SisePuede/src/java3/Conexion.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package java3;
import java.sql.*;
/**
 *
 * @author Gabo
 */
public class Conexion {
    
    Connection cn = null;
    
    public Connection conectar(String base){
        
        String host ="jdbc:mysql://localhost/"+base;
        String user = "root";
        String psw ="";
        
        try {
            cn = DriverManager.getConnection(host,user,psw);
        } catch (SQLException e) {
            System.out.println(e);
        }
        return cn;
    }
}
